app/mesh/1.0.x/installation/docker.md

```diff
@@ -26,15 +26,15 @@ You have a license for {{site.mesh_product_name}}.
 {{site.mesh_product_name}} provides the following Docker images for all of its
 executables:
 
-* **kuma-cp**: at `kong-docker-kong-mesh-docker.bintray.io/kuma-cp:{{page.kong_versions[0].version}}`
-* **kuma-dp**: at `kong-docker-kong-mesh-docker.bintray.io/kuma-dp:{{page.kong_versions[0].version}}`
-* **kumactl**: at `kong-docker-kong-mesh-docker.bintray.io/kumactl:{{page.kong_versions[0].version}}`
-* **kuma-prometheus-sd**: at `kong-docker-kong-mesh-docker.bintray.io/kuma-prometheus-sd:{{page.kong_versions[0].version}}`
+* **kuma-cp**: at `docker.io/kong/kuma-cp:{{page.kong_versions[0].version}}`
+* **kuma-dp**: at `docker.io/kong/kuma-dp:{{page.kong_versions[0].version}}`
+* **kumactl**: at `docker.io/kong/kumactl:{{page.kong_versions[0].version}}`
+* **kuma-prometheus-sd**: at `docker.io/kong/kuma-prometheus-sd:{{page.kong_versions[0].version}}`
 
 `docker pull` each image that you need. For example:
 
 ```sh
-$ docker pull kong-docker-kong-mesh-docker.bintray.io/kuma-cp:{{page.kong_versions[0].version}}
+$ docker pull docker.io/kong/kuma-cp:{{page.kong_versions[0].version}}
```

Comment:
```suggestion
$ docker pull kong/kuma-cp:{{page.kong_versions[0].version}}
```
Since it's hosted on Docker Hub, and we called that out above, don't need to include the registry path.
(will need to apply these edits to other versions as well).